views/data.ejs

```diff
@@ -0,0 +1,54 @@
+<%- include("partials/head.ejs") %> 
+<%- include("partials/header.ejs") %>
+   <h2>Sky Events List</h2>
+    <table id="data-table">
+      <tr id="data-table-header">
+        <th>Event</th>
+        <th>Name</th>
+        <th>When</th>
+        <th>Description</th>
+        <th colspan="2"></th>
+      </tr>
+      
+      <% 
+       let dataId = ""
+      if (data && data.length) { 
+        data.forEach((d) => { 
+          dataId = d._id.toString()
+        let showDate = "" 
+        if (d.date !== null) {
+        showDate = d.date.toDateString().slice(4,15)
```

Comment:
Use the Date.toLocaleDateString method here because it's showing the wrong day, it's showing one day before the day I chose.
Example:
```
showDate = new Date(d.date).toLocaleDateString(undefined, {
          timeZone: "UTC"
        });
```